README.md

```diff
@@ -1,24 +1,90 @@
-# README
+# Rails project: Happy blog
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> In this project, I created a budget calculator website, which has categories of budgets in which you can enter transaction cost for each sub-group in other to get the total budget price in that particular budget category.
 
```

Comment:
- [ ] [OPTIONAL] You have an awesome design however I think it would be better to add the screenshot of the app in your README file